src/sentry/auth_v2/endpoints/csrf.py

```diff
@@ -40,7 +44,7 @@ class CsrfTokenEndpoint(AuthV2Endpoint):
         },
     )
     @method_decorator(ensure_csrf_cookie)
-    def get(self, request, *args, **kwargs):
+    def get(self, request: Request, *args: Any, **kwargs: Any) -> Response:
```

Comment:
does this actually take args / kwargs?  if there's no url parameters I believe you can / should leave these out